README.md

```diff
@@ -52,13 +52,36 @@ export default {
 
 **Note:** Nuxt >= 2.15.0 is required
 
+## 💡 Vite Config
+
+You can pass Vite configurations and plugins into the `vite` entry of `nuxt.config`
+
+```js
+import ViteWindiCSS from 'vite-plugin-windicss'
+
+// nuxt.config
+export default {
+  buildModules: [
+    'nuxt-vite'
+  ],
+  vite: {
+    plugins: [
+      ViteWindiCSS()
+    ],
+    /* ... */
+  }
+}
+```
+
+More details refer to [Vite's documentations](https://vitejs.dev/config/).
+
```

Comment:
Maybe we can also mention about additional vite.vue config and look nk to it's docs?